src/main.js

```diff
@@ -0,0 +1,105 @@
+import './style/main.scss'
+import React from 'react'
+import ReactDom from 'react-dom'
+import superagent from 'superagent'
+
+const API_URI = 'https://www.reddit.com/r'
+
+class RedditForm extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      data: '',
+    }
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleRedditQuery = this.handleRedditQuery.bind(this)
+  }
+
+  handleRedditQuery(e){
+    this.setState({data: e.target.value})
+  }
+
+  handleSubmit(e){
+    e.preventDefault()
+    this.props.redditSelect(this.state.data)
+  }
+
+  render(){
+    return (
+      <form onSubmit={this.handleSubmit} >
+        <input
+          type='text'
+          data='postTitle'
+          placeholder='Post Title'
+          value={this.state.data}
+          onChange={this.handleRedditQuery}
+          />
+      </form>
+    );
+  }
+}
+
+class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      redditTitle: null,
+      redditLookup: {},
+      redditSelected: null,
+      redditDataError: null,
+      }
+
+    this.redditSelect = this.redditSelect.bind(this)
+    }
+    componentDidUpdate(){
+      console.log('___STATE___', this.state);
+    }
+
+  redditSelect(data) {
+      console.log('about to get');
+      superagent.get(`${API_URI}/${data}.json`)
+      .then(res => {
+        console.log('selected post', res.body);
+        this.setState({
+          redditTitle: data,
+          redditSelected: res.body.kind,
```

Comment:
Change 'res.body.kind' to 'res.body.data.children' so you can get the 25 objects you need from Reddit. While 'res.body.kind' will only return "Listed".